public/sass/partials/_accent-bar.scss

```diff
@@ -0,0 +1,30 @@
+// these move into a Typography or Font file
+h1 {
+  padding-bottom: 1em;  // REQUIRED for accent bar do not remove
+  position: relative;   // REQUIRED for accent bar do not remove
+  &.center-align { //this is purely for sample purposes!
+    text-align: center;
+  }
+  .accent-bar-L {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+  .accent-bar-C {
+    position: absolute;
+    bottom: 0;
+    left: 50% - 3%;
+  }
+}
+// these stay here
+%accent-bar {
+  background-color: $color-5;
+  height: 0.1em;
+  width: 6%;
+}
+.accent-bar-L {
```

Comment:
why the different names? If they extend the same code, why not use a name that is a little less specific?